apiserver/facades/controller/migrationtarget/migrationtarget_test.go

```diff
@@ -35,9 +39,10 @@ import (
 type Suite struct {
 	statetesting.StateSuite
 	resources  *common.Resources
-	authorizer apiservertesting.FakeAuthorizer
+	authorizer *apiservertesting.FakeAuthorizer
```

Comment:
why this change?
when we store a facade context struct and not a pointer (for example)

Reply:
I needed to be able to change it in a test and have the change be visible in the constructed facade. When it wasn't a pointer then the authorizer was getting copied into the facade context, and changes made in the body of the test had no effect.